tapeagents/batch.py

```diff
@@ -9,9 +9,11 @@
 
 from pydantic import BaseModel
 
+from tapeagents.llms import LLMEvent, LLMStream, TrainableLLM
+
 from .agent import Agent, Annotator, ObservationMaker
```

Comment:
Lets make all framework imports here absolute, like `from tapeagents.xxxxx`. We should not have both absolute and relative import from the same folder in a single file